rust/ql/lib/codeql/rust/elements/DerefExpr.qll

```diff
@@ -0,0 +1,13 @@
+/**
+ * Provides classes for deref expressions (`*`).
+ */
+
+private import codeql.rust.elements.PrefixExpr
+private import codeql.rust.elements.Operation
+
+/**
+ * A dereference expression, `*`.
```

Comment:
```suggestion
 * A dereference expression, the prefix operator `*`.
```
Or something of that sort. I feel that `*` just by itself is not precise enough as there is also the binary `*` multiplication.